frontend/resourceadm/components/AccessListDetails/AccessListDetail.test.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { render as rtlRender, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { act } from 'react-dom/test-utils';
+import { textMock } from '../../../testing/mocks/i18nMock';
+import { queriesMock } from 'app-shared/mocks/queriesMock';
+import { AccessListDetail, AccessListDetailProps } from './AccessListDetail';
+import { ServicesContextProps, ServicesContextProvider } from 'app-shared/contexts/ServicesContext';
+import { createQueryClientMock } from 'app-shared/mocks/queryClientMock';
+
+const mockedNavigate = jest.fn();
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useNavigate: () => mockedNavigate,
+}));
+
+const testOrg = 'ttd';
+const testEnv = 'tt02';
+const testListIdentifier = 'listid';
+
+const defaultProps = {
+  org: testOrg,
+  env: testEnv,
+  list: {
+    env: testEnv,
+    identifier: testListIdentifier,
+    name: 'Test-list',
+    description: 'This is a description',
+    members: [],
+  },
+  backUrl: '/listadmin',
+};
+
+describe('AccessListDetail', () => {
+  it('should call service to update name', async () => {
```

Comment:
I would suggest adding this line on line 36 to make sure we clean up the tests are they are run through: 
```tsx
  afterEach(jest.clearAllMocks);
```